src/components/SignInForm.js

```diff
@@ -0,0 +1,107 @@
+import React, { useContext, useState } from "react";
+import { useHistory } from "react-router-dom";
+import { AuthContext } from "../App";
+import { UserBase } from "../UserBase";
+import Linked from "./Linked";
+
+import Form from "./Form";
+import Input from "./Input";
+import Button from "./Button";
+import SeparForm from "./SeparForm";
+
+const SignInForm = () => {
+  const { setCurrentUser, currentUser } = useContext(AuthContext);
+  const history = useHistory();
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  function handleUsernameChange(e) {
+    setUsername(e.target.value);
+  }
+
+  function handlePasswordChange(e) {
+    setPassword(e.target.value);
+  }
+
+  function submitForm() {
+    console.log(username);
+    if (!username) {
+      return alert("Your need to enter a Username");
+    } else if (!password) {
+      return alert("Your need to enter a Password");
+    } else if (UserBase.some((el) => el.pseudo === username)) {
+      for (let i = 0; i < UserBase.length; i++) {
+        if (UserBase[i].pseudo.indexOf(username) !== -1) {
+          if (UserBase[i].password.indexOf(password) !== -1) {
+            setCurrentUser(username);
+            //history.push("/game-list");
+          } else {
+            return alert("password is not correct");
+          }
+        }
+      } //Ferme for
+    } else {
+      alert("user is not defined");
+    } // Ferme else if some
+  }
+
+  if (currentUser) {
+    const detailsUser = UserBase.find(el => el.pseudo === currentUser);
+    return (
+      <Form>
```

Comment:
You put home in Form component and in sign in ??

Reply:
It was for the test. I will change the root, when everybody will push their code to make goods components Links (idem for the loggedname in the header)

Reply:
pr ok